Command/CreateSiteCommand.php

```diff
@@ -102,9 +109,26 @@ public function execute(InputInterface $input, OutputInterface $output)
   <info>enabled</info> :  <info>from</info> {$info_enabledFrom} => <info>to</info> {$info_enabledTo}
 
 INFO
-);
+        );
 
-        if ($input->getOption('no-confirmation') || $dialog->askConfirmation($output, 'Confirm site creation ?', false)) {
+        /*
+         * NEXT_MAJOR: remove if statement and use the question helper only (true value of if) when dropping sf < 2.5
+         */
+        if ($this->getHelperSet()->has('question')) {
+            $question = new \Symfony\Component\Console\Question\ConfirmationQuestion(
+                'Confirm site creation (Y/N)',
+                false,
+                '/^(y|j)/i'
```

Comment:
Wel thats a funny story but i was looking the symfony 3 explanation and it had it as an example. But thought it made sense since symfony is in essence a multi langual platform, and in dutch for instance is 'ja'. But if ya think it shouldnt be there i dont mind taking it away